src/Features/CSharp/Portable/SignatureHelp/ObjectCreationExpressionSignatureHelpProvider_DelegateType.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.CodeAnalysis.CSharp.SignatureHelp
 {
     internal partial class ObjectCreationExpressionSignatureHelpProvider
     {
-        private IList<SignatureHelpItem> GetDelegateTypeConstructors(
+        private (IList<SignatureHelpItem> Items, int? SelectedItem) GetDelegateTypeConstructors(
```

Comment:
:bulb: Tuple element names should be camelCase. We are working on a code style to enforce/fix this, but addressing it earlier will make the cleanup pass less work. :smile: